integrated_channels/sap_success_factors/exporters/content_metadata.py

```diff
@@ -109,6 +111,24 @@ def transform_description(self, content_metadata_item):
 
         return description_with_locales
 
+    def transform_estimated_hours(self, content_metadata_item):
```

Comment:
I think we should move this method in the parent class and inherit it from there for SAP and Cornerstone.

Reply:
looks like it is already in the parent class? some of the functionality is duplicated so we could probably call the parent class to do the calculation bit that is in common but keep this one here since part of it seems specific to SAP

Reply:
Agree with @brittneyexline . I have moved out the duplicated code for transforming the course duration.